docker-compose.yml

```diff
@@ -75,8 +76,11 @@ services:
       REDCAP_API_URL: ${REDCAP_API_URL}
       OTP_ENABLED: ${OTP_ENABLED}
       CTRL_ADMIN_EMAIL: ${CTRL_ADMIN_EMAIL}
+      RAILS_MASTER_KEY: ${RAILS_MASTER_KEY}
+      ALLOWED_HOSTS: ${ALLOWED_HOSTS}
 
   selenium:
+    profiles: [test]
```

Comment:
Adds [docker compose profiles](https://docs.docker.com/compose/profiles/) so that test containers don't need to be run in non-test situations.